server/events/vcs/lyft/mergeability.go

```diff
@@ -33,6 +33,28 @@ func (f NameStateFilter) Filter(statuses []*github.RepoStatus) []*github.RepoSta
 	return filtered
 }
 
+type NameCheckFilter struct {
+	checkName string
+	status    string
+}
+
+func NewSQCheckFilter() NameCheckFilter {
+	return NameCheckFilter{checkName: SubmitQueueReadinessContext, status: "queued"}
```

Comment:
Hmm should it be `queued` or `in-progress`? The case this solves is when apply checks are in the failed state and we want to be able to run apply again while still making sure the PR is mergeable.  

It wouldn't be mergeable based on the api definition because sq-ready-merge wouldn't be green given the apply check errors so we filter out sq-ready-merge and make sure the rest of the checks are all passing before we allow the apply

Reply:
Based on offline discussion: buildnotify doesn't use in-progress so this is a moot point